exercises/prime-factors/PrimeFactorsTest.cs

```diff
@@ -7,42 +7,42 @@ public class PrimeFactorsTest
     [Fact]
     public void No_factors()
     {
-        Assert.Empty(PrimeFactors.Factors(1));
+        Assert.Empty(PrimeFactors.Factors(1L));
```

Comment:
I've now made it clear that each input value is actually a long value. Previously, an implicit conversion was taking place, but this is more clear I feel.